### PR TITLE
Added a couple of devices / modules

### PR DESCRIPTION
**Pull request by MrCamby**
As the title says I have added a few devices and modules.

Devices
- Check Point SG7000-PLUS
- Check Point SG16200-PLUS
- Gude EMC Professional 3011
- HPE DX380 Gen10

Modules
- Check Point CPAC-8-1C-C
- Check Point CPAP-TR-100SWDM4
- Check Point CPAP-TR-10SR-C
- Cisco C9200-NM-4X
- HPE 535T
- HPE 640FLR-SFP28
- HPE 640SFP28

### CODE REVIEW

File: device-types/Check Point/SG7000-PLUS.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  maximum_draw: null
+  allocated_draw: null
+  label: ''
+  description: ''
+- name: PSU-2
+  type: iec-60320-c14
+  maximum_draw: null
+  allocated_draw: null
+  label: ''
+  description: ''
```
Comment (DanSheps) [inline device-types/Check Point/SG7000-PLUS.yaml:32]: If these are modular at all, they should be broken out to modules

File: device-types/Check Point/SG16200-PLUS.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  maximum_draw: 1300
+  allocated_draw: null
+  label: ''
+  description: ''
+- name: PSU-2
+  type: iec-60320-c14
+  maximum_draw: 1300
+  allocated_draw: null
+  label: ''
+  description: ''
```
Comment (DanSheps) [inline device-types/Check Point/SG16200-PLUS.yaml:32]: If these are modular, they should be broken out into modules

File: device-types/Gude/emc-professional-3011.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  maximum_draw: 5
+  allocated_draw: 5
+  label: ''
+  description: ''
+interfaces:
+- name: LAN
+  type: 100base-tx
+  mgmt_only: false
+  label: ''
+  description: ''
```
Comment (DanSheps) [inline device-types/Gude/emc-professional-3011.yaml:28]: Please separate this out into a different PR

File: device-types/HPE/DX380_Gen10.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+airflow: front-to-rear
+power-ports:
+- name: PSU1
+  type: iec-60320-c14
+  maximum_draw: 1600
+  allocated_draw: null
+- name: PSU2
+  type: iec-60320-c14
+  maximum_draw: 1600
+  allocated_draw: null
```
Comment (DanSheps) [inline device-types/HPE/DX380_Gen10.yaml:16]: Should be modular

File: device-types/HPE/DX380_Gen10.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+- name: iLO
+  type: 1000base-t
+  mgmt_only: true
```
Comment (DanSheps) [inline device-types/HPE/DX380_Gen10.yaml:19]: Should be a interface not a power port

File: device-types/HPE/DX380_Gen10.yaml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+---
+manufacturer: HPE
+model: DX380 Gen10
+slug: hpe_dx380_g10
```
Comment (DanSheps) [inline device-types/HPE/DX380_Gen10.yaml:4]: Separate out into a different PR

Looks like you may be missing your module for PCI risers as well

File: device-types/HPE/ProLiant_DL360_Gen10.yaml [hunk 7]
```diff
@@ -4,6 +4,8 @@ model: ProLiant DL360 Gen10
 slug: hpe_dl360_g10
 u_height: 1
 is_full_depth: true
+airflow: front-to-rear
+comments: '[HPE DL360 Gen10 Quickspecs](https://www.hpe.com/psnow/doc/a00008159enw.pdf)'
```
Comment (DanSheps) [inline device-types/HPE/ProLiant_DL360_Gen10.yaml:8]: Separate out into different PR

File: module-types/Cisco/C9200-NM-4X.yaml [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+- name: TenGigabitEthernet1/{module}/3
+  type: 10gbase-x-sfpp
+  mgmt_only: false
+  label: ''
+  description: ''
+- name: TenGigabitEthernet1/{module}/4
+  type: 10gbase-x-sfpp
+  mgmt_only: false
+  label: ''
+  description: ''
```
Comment (DanSheps) [inline module-types/Cisco/C9200-NM-4X.yaml:26]: Different PR please

File: module-types/HPE/535T.yaml [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+model: Ethernet 10Gb 2-port 535T Adapter
+part_number: 813661-B21
+comments: HPE Ethernet 10Gb 2-port 535T Adapter
+interfaces:
+- name: '{module}-RJ45_1'
+  type: 10gbase-t
+  mgmt_only: false
+- name: '{module}-RJ45_2'
+  type: 10gbase-t
+  mgmt_only: false
```
Comment (DanSheps) [inline module-types/HPE/535T.yaml:12]: Different PR please

File: module-types/HPE/640FLR-SFP28.yaml [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+model: Ethernet 10/25Gb 2-port 640FLR-SFP28 Adapter (LOM)
+part_number: 817749-B21
+comments: "HPE Ethernet 10/25Gb 2-port 640FLR-SFP28 Adapter\r\nLOM_Slot"
+interfaces:
+- name: '{module}-SFP28_1'
+  type: 25gbase-x-sfp28
+  mgmt_only: false
+- name: '{module}-SFP28_2'
+  type: 25gbase-x-sfp28
+  mgmt_only: false
```
Comment (DanSheps) [inline module-types/HPE/640FLR-SFP28.yaml:12]: Different PR please

File: module-types/HPE/640SFP28.yaml [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+model: Ethernet 10/25Gb 2-port 640SFP28 Adapter
+part_number: 817753-B21
+comments: HPE Ethernet 10/25Gb 2-port 640SFP28 Adapter
+interfaces:
+- name: '{module}-SFP28_1'
+  type: 25gbase-x-sfp28
+  mgmt_only: false
+- name: '{module}-SFP28_2'
+  type: 25gbase-x-sfp28
+  mgmt_only: false
```
Comment (DanSheps) [inline module-types/HPE/640SFP28.yaml:12]: Different PR please

**Review by DanSheps — CHANGES_REQUESTED** [review]
Could you please break some of these out into separate PR's?

It makes it easier to review.

**Comment by danner26** [comment]
Do you still need this? @MrCamby

**Pull request closed without merging**